ui/signup.html

```diff
@@ -60,7 +60,7 @@ <h3 id="signup-title-container"><!----></h3>
                         <h4>__MSG__CREATE_A_NEW_ACCOUNT_COLON__</h4>
                         <form id="signup-createaccount-form" class="form-horizontal" role="form">
                             {if invitationInfo.email}
-                                <input type="hidden" id="signup-createaccount-email" name="email" class="form-control required email maxlength-short" value="${invitationInfo.email|encodeForHTMLAttribute}" />
+                                <input type="hidden" id="signup-createaccount-email" name="email" class="form-control required email maxlength-short emaildomain" value="${invitationInfo.email|encodeForHTMLAttribute}" />
```

Comment:
Do we want all of the validation rules on the invitation email? If the validation fails for some reasons (e.g. domain was changed), the user would have no idea as to why it fails